### ISSUE TITLE
Handle inrush current wrt. OCP

### ISSUE THREAD

**Issue by kanflo** [kanflo/opendps#6]
For resistors drawing, say, 400mA, the current limit needs to be set to 600mA or the OCP will kick in immediately when enabling power.

The inrush current needs to be handled, eg. by accepting inrush current > I_limit for a short period of time.

**Comment by vogelfreiheit** [comment]
Does this refer to the initial spike when turning the output on? I observed a spike of 4V with output set to 5.20v initially which could already damage/toast some sensitive components.

**Comment by kanflo** [comment]
Yes, I have seen the spikes too. Someday I will look into this, perhaps a ramp of a few ms could solve this.

**Comment by Xenoamor** [comment]
I was thinking of this problem just recently. Perhaps having a fixed period of ~30ms and then in the settings you can adjust the percentage overcurrent it will allow for that period
